core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -409,17 +433,14 @@ void core_mmu_get_user_pgdir(struct core_mmu_table_info *pgd_info)
 	void *tbl = (void *)core_mmu_get_ul1_ttb_va();
 
 	core_mmu_set_info_table(pgd_info, 1, 0, tbl);
-	pgd_info->num_entries = TEE_MMU_UL1_NUM_ENTRIES;
+	pgd_info->num_entries = NUM_UL1_ENTRIES;
 }
 
 void core_mmu_create_user_map(struct user_ta_ctx *utc,
 			      struct core_mmu_user_map *map)
 {
 	struct core_mmu_table_info dir_info;
 
-	COMPILE_TIME_ASSERT(sizeof(uint32_t) * TEE_MMU_L2_NUM_ENTRIES ==
```

Comment:
Please keep this assert in some form, it's used to guarantee that the pgt_cache code is in sync with this code.